skyportal/model_util.py

```diff
@@ -79,6 +79,9 @@ def setup_permissions():
     """Create default ACLs/Roles needed by application.
 
     If a given ACL or Role already exists, it will be skipped."""
+    all_acl_ids = ['Become user', 'Comment', 'Manage users', 'Manage sources',
```

Comment:
@profjsb this is defined as a global above. Also, currently only the super admin can classify unless you add it to the ACLs for other roles above